cli_tools_tests/e2e/gce_ovf_import/test_suites/ovf_machine_image_import/ovf_machine_image_import_test_suite.go

```diff
@@ -85,12 +85,15 @@ func TestSuite(
 			testSuiteName, fmt.Sprintf("[%v] %v", testType, "Windows 2012 R2 two disks, Network setting (path)"))
 		machineImageImportStorageLocationTestCase := junitxml.NewTestCase(
 			testSuiteName, fmt.Sprintf("[%v] %v", testType, "Centos 7.4, Storage location"))
+		machineImageImportNetworkTagsTestCase := junitxml.NewTestCase(
```

Comment:
Same here, merge with an existing test.

Reply:
done! Added tags to the "network setting" tests, since the tags are network related.